frontend/src/AppRoutes/index.tsx

```diff
@@ -178,23 +178,25 @@ function App(): JSX.Element {
 	}, [pathname]);
 
 	useEffect(() => {
-		try {
-			const isThemeAnalyticsSent = getLocalStorageApi(
-				LOCALSTORAGE.THEME_ANALYTICS,
-			);
-			if (!isThemeAnalyticsSent) {
-				trackEvent('Theme Analytics', {
-					theme: isDarkMode ? THEME_MODE.DARK : THEME_MODE.LIGHT,
-					user: pick(user, ['email', 'userId', 'name']),
-					org,
-				});
-				setLocalStorageApi(LOCALSTORAGE.THEME_ANALYTICS, 'true');
+		if (user && user?.email && user?.userId && user?.name) {
```

Comment:
added the check to send the event only after the user is fetched. and set the local state to not send further events